components/split-view/private/list-item-with-Content.jsx

```diff
@@ -0,0 +1,130 @@
+/* Copyright (c) 2015-present, salesforce.com, inc. All rights reserved */
+/* Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license */
+
+/* eslint-disable class-methods-use-this */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+export const DISPLAY_NAME = 'SLDSSplitViewListItemWithContent';
+
+const propsTypes = {
+	/**
+	 * **Assistive text for accessibility**
+	 * * `unreadItem`: The unread indicator.
+	 */
+	assistiveText: PropTypes.shape({
+		unreadItem: PropTypes.string
+	}),
+	/**
+	 * Item to be displayed
+	 */
+	item: PropTypes.object.isRequired,
+	/**
+	 * Allows multiple item to be selection
+	 */
+	multiple: PropTypes.bool,
+	/**
+	 * Shows the item as `focused`.
+	 */
+	isFocused: PropTypes.bool.isRequired,
+	/**
+	 * Shows the item as `selected`.
+	 */
+	isSelected: PropTypes.bool.isRequired,
+	/**
+	 * Shows the item as `unread`.
+	 */
+	isUnread: PropTypes.bool,
+	/**
+	 * **Event Callbacks**
+	 * * `onClick`: Called when the item is clicked.
+	 * * * Event
+	 * * * Meta data
+	 * * * * `item`: The original item.
+	 * * * * `isSelected`: Is the item selected.
+	 * * * * `isUnread`: Is the item unread.
```

Comment:
Does ` * * * *` create a valid list in markdown?

Reply:
this is what it does in md.

**Event Callbacks**
* `onClick`: Called when the item is clicked.
* * Event
* * Meta data
* * * `item`: The original item.
* * * `isSelected`: Is the item selected.
* * * `isUnread`: Is the item unread.